tournamentcontrol/competition/models.py

```diff
@@ -513,6 +517,17 @@ class Season(AdminUrlMixin, RankImportanceMixin, OrderedSitemapNode):
     live_stream_token_uri = models.URLField(null=True)
     live_stream_scopes = PG.ArrayField(models.CharField(max_length=200), null=True)
     live_stream_thumbnail = models.URLField(blank=True, null=True)
+    thumbnail_image = models.BinaryField(
+        blank=True,
+        null=True,
+        help_text="Binary data for thumbnail image to be used for YouTube videos",
+    )
+    thumbnail_image_mimetype = models.CharField(
```

Comment:
Do not store the MIME type, we can easily calculate and we won't be using this unless we are also working with the image data.